benchmark/C/Savina/src/parallelism/RadixSort.lf

```diff
@@ -9,9 +9,13 @@
  */
 
 
-target C;
-
-
+target C {
+    /* [[[cog
+    cog.outl(f'threads: {threads}')
+    ]]] */
+    threads: 24 
+    // [[[end]]]
+};
 
 reactor IntSourceReactor(numValues:int(100000), maxValue:{=long=}({=1l << 60=}), seed:{=long=}(2048)) {
 
```

Comment:
The comment refers to the code below. The `nextLong` function below looks wrong to me, as it does not preserve state and thus always produces the same number. There is a random number generator defined in `include/PseudoRandom.h` which should be used instead.

Reply:
If I just delete that code for nextLong and include PseudoRandom.h, will that suffice?  The nextlong function in PseudoRandom.h is identical to the nextLong function that was in this preamble, though.

Reply:
The difference is that the random number generator in PseudoRandom.h has internal state that is updated on each call to `nextLong`. Thus, it will produce different numbers on each call, whereas the implementation here should always produce the same number. In order to call `nextLong` from `PseudoRandom.h` you also need to instantiate the `PseudoRandom` struct and then initialize it with the seed in a startup reaction.